js/app.js

```diff
@@ -114,71 +114,76 @@ cardItem.forEach((item, j) => {
   callCardBody(j);
 });
 
-const butn = document.querySelectorAll('.btn');
-butn.forEach((but) => {
-  but.addEventListener('click', (e) => {
-    cardItem.forEach((project) => {
-      if (e.target.getAttribute('id') === project.button) {
-        proName = project.pName;
-        proDescription = project.pDescription;
-        proFeatures = project.pFeatures;
-        proImage = project.pImage;
-        proTech = project.pTech;
-        // console.log(t);
-      }
-    });
-    const cardContent = `<div><div class="desk_popup_title"><h2 class="desk_title">${proName}</h2><div class="desk_popup_close"><div class="bar1"></div><div class="bar2"></div></div></div>
-        <ul class="dev desk_dev">
-        <li class="upper dev_item font_p">${proFeatures[0]}</li>
-    <li class="dev_item"><div class="seperator"></div></li>
-    <li class="dev_item font_p">${proFeatures[1]}</li>
-    <li class="dev_item"><div class="seperator"></div></li>
-    <li class="dev_item font_p">${proFeatures[2]}</li>
-  
-  </ul>
-  <div class="desk_popup_image"> <img class="project_image" src="${proImage[0]}" alt="project 1" />
-  <img
-    class="project_image_des"
-    src="${proImage[1]}"
-    alt="project_1"
-  />
-  </div>
-  <div class="desk_popup_detail">
-  <div class="desk_popup_info">
-  <p class="project_details font_p">
-  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.  
-  </p>
-  </div>
-  <div class="desk_link_con">
-  <ul><li><ul class="api mob_menu">
-  <li class="api_item font_p mob_item">${proTech[0]}</li>
-  <li class="api_item font_p mob_item">${proTech[1]}</li>
-  <li class="api_item font_p mob_item">${proTech[2]}</li>
-  
+const overlay = document.querySelector('.j-popup');
+const whiteBG = document.querySelector('.whitebg');
+overlay.classList.toggle('active');
+whiteBG.classList.toggle('active');
+
+const cardContent = `<div><div class="desk_popup_title"><h2 class="desk_title">${proName}</h2><div class="desk_popup_close"><div class="bar1"></div><div class="bar2"></div></div></div>
```

Comment:
- [ ] The popup is generating the same content for every project, kindly fix this so that when the user clicks on the ``` See Project``` button a unique popup with the proper image and information is displayed 

![Screenshot_20220514_015523](https://user-images.githubusercontent.com/25227022/168405443-bb4c2e39-3274-46db-8c66-579f6820c231.png)